### PR TITLE
octanifies seq block session manager component.

### PR DESCRIPTION
**Pull request by stopfstedt**
Not using Resources here is intentional, the `*load()` task with the `did-insert/did-update` modifiers seems to be better pattern for initializing/reloading state - we're operating on copies of the passed-in data.

### CODE REVIEW

File: tests/integration/components/curriculum-inventory/sequence-block-session-manager-test.js [hunk 1]
```diff
@@ -1,736 +1,514 @@
...
 import hbs from 'htmlbars-inline-precompile';
-
-const { resolve } = RSVP;
+import { component } from 'ilios/tests/pages/components/curriculum-inventory/sequence-block-session-manager';
 
 module(
   'Integration | Component | curriculum-inventory/sequence-block-session-manager',
   function (hooks) {
     setupRenderingTest(hooks);
+    setupMirage(hooks);
```
Comment (jrjohnson) [inline tests/integration/components/curriculum-inventory/sequence-block-session-manager-test.js:13]: Still makes me happy every time we update one of these!

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: app/components/curriculum-inventory/sequence-block-session-manager.js [hunk 2]
```diff
@@ -1,185 +1,145 @@
...
+  }
+
+  @action
+  close() {
+    this.args.cancel();
+  }
+
+  @restartableTask
+  *load() {
+    this.linkedSessions = (yield this.args.sequenceBlock.sessions).toArray();
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply app/components/curriculum-inventory/sequence-block-session-manager.js:136]: :heavy_check_mark:

File: .lint-todo/c07378a47dc7bd8a54d2c26205534f9c1160f3c5/7040d01a.json [hunk 3]
```diff
@@ -1 +1 @@
-{"engine":"ember-template-lint","filePath":"app/components/curriculum-inventory/sequence-block-session-manager.hbs","ruleId":"require-input-label","line":83,"column":16,"createdDate":1615276800000,"warnDate":1617865200000,"errorDate":1620457200000}
+{"engine":"ember-template-lint","filePath":"app/components/curriculum-inventory/sequence-block-session-manager.hbs","ruleId":"require-input-label","line":34,"column":14,"createdDate":1616137200000,"warnDate":1618729200000,"errorDate":1621321200000}
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply .lint-todo/c07378a47dc7bd8a54d2c26205534f9c1160f3c5/7040d01a.json]: using `aria-labelledby`, see https://www.w3.org/TR/WCAG20-TECHS/ARIA9.html#ARIA9-ex2

[the remark this answers is not in the document]
Comment (stopfstedt) [reply .lint-todo/c07378a47dc7bd8a54d2c26205534f9c1160f3c5/7040d01a.json]: :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

File: .lint-todo/c07378a47dc7bd8a54d2c26205534f9c1160f3c5/7040d01a.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jrjohnson) [reply .lint-todo/c07378a47dc7bd8a54d2c26205534f9c1160f3c5/7040d01a.json]: Good solution. You could also just `aria-label={{t "something"}}` and ignore the label that is there already. I'm not sure if that is a good solution for people using assistive technology though, yours is probably better.

**Pull request merged**